subs/pantry/src/Pantry/Types.hs

```diff
@@ -904,6 +945,13 @@ mkSafeFilePath t = do
 
   Just $ SafeFilePath t
 
+hpackSafeFilePath :: SafeFilePath
```

Comment:
Please add a Haddock explaining what this value is.

Reply:
Added.